core/src/main/java/org/infinispan/interceptors/CacheLoaderInterceptor.java

```diff
@@ -415,7 +444,12 @@ private boolean skipLoad(FlagAffectedCommand cmd, Object key, InvocationContext
       }
 
       boolean skip;
-      if (cmd instanceof WriteCommand) {
+      if (cmd instanceof AbstractWriteKeyCommand || cmd instanceof AbstractWriteManyCommand) {
```

Comment:
Since the line 452 (formerly 418) uses instanceof, I thought I might as well add some more tests like this. I'm not etirely confident that testing for Abstract* classes is the way to go - maybe I should add a marker interface FunctionalCommand?